Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+    ln_s buildpath, buildpath/"src/github.com/bitrise-io/bitrise"
+
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "_bin/bitrise"
+
+    bin.install "_bin/bitrise"
+  end
+
+  test do
+    output = shell_output(bin/"bitrise --version")
```

Comment:
I'm OK with this solution, just to be sure:

1. If we do this (to download the binary or source of the other two tools) and we update `bitrise` in the future which will require newer versions of these tools it will update the tools automatically as well, right? Even if a previous version of `envman` or `stepman` is already available?
2. If we have to build the other two tools from source: can you point to another Formula which does this and is written in Go or uses another Go tool, like we do? It wasn't exactly simple to prepare a proper Go environment for the build, it would help a lot to see another Formula which builds it's Go dependency.

Thank you!